lib/SILOptimizer/IPO/LetPropertiesOpts.cpp

```diff
@@ -210,22 +210,18 @@ void LetPropertiesOpt::optimizeLetPropertyAccess(VarDecl *Property,
       // Replace the access to a let property by the value
       // computed by this initializer.
       SILValue clonedInit = cloneInitAt(proj);
-      SILBuilderWithScope B(proj);
       for (auto UI = proj->use_begin(), E = proj->use_end(); UI != E;) {
         auto *User = UI->getUser();
         ++UI;
 
-        if (isIncidentalUse(User))
-          continue;
-
         // A nested begin_access will be mapped as a separate "Load".
         if (isa<BeginAccessInst>(User))
           continue;
```

Comment:
Is it correct to not call canReplaceLoadSequence for a begin_access?

Reply:
Yes, because of the way LetPropertiesOpts is written, the operand the begin access will appear as a separate 'load' in the list. So we'll visit it again later.